frontend/src/app/organization/organization-list/organization-list.component.ts

```diff
@@ -0,0 +1,17 @@
+import { Component } from '@angular/core';
+import { OrganizationService } from '../organization.service';
+import { Organization } from '../organization.model';
+
+@Component({
+  selector: 'volontulo-organization-list',
+  templateUrl: './organization-list.component.html',
+  providers: [OrganizationService]
+})
+export class OrganizationListComponent {
+
+  organizations: Organization[];
+
+  constructor(private orgService: OrganizationService) {
+    this.orgService.query().subscribe((orgs: Organization[]) => this.organizations = orgs)
```

Comment:
Whereas this logic in the constructor will work, it's not really semantically correct. This logic is related to component's lifecycle, not to JS object creation.

Better: the components implements `OnInit` interface and the logic is moved to `ngOnInit` method, constructor only used to inject dependencies here.

Reply:
When changed, a there might be a side effect problem (I'm not entirely sure without running it) cause your component will fail, because `organizations.length` will throw TypeError (no length on undefined). In fact, that would be good, you would have a better control on the loading process.